src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs

```diff
@@ -37,5 +37,19 @@ public bool LoadUserProfile
             get { return _loadUserProfile; }
             set { _loadUserProfile = value; }
         }
+
+        // CoreCLR can't correctly support UseShellExecute=true for the following reasons
+        // 1. ShellExecuteEx is not supported on onecore.
```

Comment:
Do we have any ability to "light up" this functionality if you're not limited to running on onecore? Or would that cause an application to fail WACK or something like that?